content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -93,6 +93,44 @@ EndpointSlices support three address types:
 * IPv6
 * FQDN (Fully Qualified Domain Name)
 
+### Conditions
+
+The EndpointSlice API stores conditions about endpoints that may be useful for consumers.
+The three conditions are `ready`, `serving`, and `terminating`.
+
+#### Ready
+
+`Ready` is a condition that maps to a pod's `Ready` condition. A running pod with the `Ready` condition set to `True`
+should have this EndpointSlice condition also set to `true`. For compatibility reasons, `ready` is NEVER `true` when a
+pod is terminating. Consumers should refer to the `serving` condition to inspect pod readiness for terminating pods.
+The only exception to this rule is for Services with `spec.publishNotReadyAddresses` set to `true`. Endpoints for these
+Services will always have `ready` condition.
+
+#### Serving
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+`Serving` is identical to the `Ready` condition, except it does not account for terminating states. Consumers of the
+EndpointSlice API should check this condition if they care about pod readiness while the pod is also terminating.
+
+{{< note >}}
+
+Although `serving` is almost identical to `ready`, it was added to not break the existing meaning of `ready`.
+It may be unexpected for existing clients if `ready` could be `true` for terminating endpoints, since historically
+terminating endpoints were never included in the Endpoints or EndpointSlice API to begin with. For this reason,
+`ready` is ALWAYS `false` for terminating endpoints, and a new condition `serving` was added in v1.20 so that clients
+can track readiness for terminating pods independent of the existing semantics for `ready`.
+
+{{< /note >}}
+
+#### Terminating
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+`Terminating` is a condition that indicates whether an endpoint is terminating. For pods, this is any pod that has a deletion
+timestamp and is slated for terminating by Kubernetes.
```

Comment:
Nit: I'm not sure we need to say "and is slated for terminating by Kubernetes". It could be confused as meaning something else must be true in addition to a deletion timestamp being set.
```suggestion
`Terminating` is a condition that indicates whether an endpoint is terminating. For pods, this is any pod that has a deletion
timestamp set.
```

Reply:
ack, will update